packages/app/src/__deprecated__/renderer/wrappers/app-update-step-modal/app-update-step-modal.component.tsx

```diff
@@ -82,17 +87,18 @@ const AppUpdateStepModal: FunctionComponent<Props> = ({
 
   return (
     <>
-      {forced ? (
```

Comment:
🟡 
I'm not sure that we should remove the force Mudita Center update logic with this PR. 

The standard and force Mudita Center should not be changed